### PR TITLE
Ensure source selection does not get hidden in overflow

### PR DESCRIPTION
**Pull request by hannahblair**
## Description

Fixes the overflow of the source selection element in video and audio. 

 
Screenshot 2023-11-06 at 19 31 32

```
import gradio as gr

with gr.Blocks() as demo:
    with gr.Row():
        input_video = gr.Video(label="Input Video1", sources=["webcam", "upload"],  interactive=True, height=400)
        output = gr.Video(label="Output Video1", height=250)

    with gr.Row():
        input_video = gr.Audio(label="Input Audio", sources=["upload", "microphone"])
        output = gr.Audio(label="Output Audio", sources=["upload", "microphone"])

if __name__ == "__main__":
    demo.launch()
```

Closes: #6258

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: js/audio/Audio.stories.svelte [hunk 1]
```diff
@@ -30,3 +30,13 @@
 		label: "Audio Recorder"
 	}}
 />
+
+<Story
```
Comment (freddyaboulton) [inline js/audio/Audio.stories.svelte:34]: 🥳

File: js/video/Video.stories.svelte [hunk 2]
```diff
@@ -50,8 +54,21 @@
...
 		width: 400
 	}}
 />
+
+<Story
+	name="Upload video"
+	args={{
+		label: "world video",
+		show_label: "true",
+		interactive: "true",
```
Comment (pngwn) [inline js/video/Video.stories.svelte:68]: These should be boolean values, not strings.

File: js/audio/Audio.stories.svelte (hunk 1 above)
Comment (pngwn) [reply js/audio/Audio.stories.svelte:34]: Does the image already do this? Is this not something that `Block` in atoms can solve?

Comment (pngwn) [reply js/audio/Audio.stories.svelte:34]: Wrong comment.

File: js/audio/interactive/InteractiveAudio.svelte [hunk 3]
```diff
@@ -204,75 +204,77 @@
...
-		{:else}
-			<AudioRecorder
-				bind:mode
-				{i18n}
-				{dispatch}
-				{dispatch_blob}
-				{waveform_settings}
-				{handle_reset_value}
-			/>
+<div>
```
[the remark this answers is not in the document]
Comment (pngwn) [reply js/audio/interactive/InteractiveAudio.svelte:207]: Meant to ask this here:

Does the image already do this? Is this not something that Block in atoms can solve?

**Review by pngwn — APPROVED** [review]
Looks great! Thanks for taking care of this @hannahblair!

**Pull request merged**